### ISSUE TITLE
Typescript compile-error (date-fns export and missing /typings)

### ISSUE THREAD

**Issue by JohannesLamberts** [dmtrKovalenko/material-ui-pickers#237]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers | 1.0.0-rc.1        |
| material-ui         | material-ui@1.0.0-beta.35        |
| React               | 16.2.0        |
| Browser             | -        |
| Platform            | WIN        |

## Steps to reproduce
1. Create a React-Project with typescript
2. Use direct import from `material-ui-pickers`
3. (do not install `date-fns`)

## Expected behavior
Compile without error.

## Actual behavior

```
Failed to compile.

./node_modules/material-ui-pickers/utils/date-fns-utils.js
Module not found: Can't resolve 'date-fns/addDays' in 'C:\Development\cms\client\node_modules\material-ui-pickers\utils'
```

I can get arount that if I replace `import { DatePicker } from 'material-ui-pickers';` by  `import DatePicker from 'material-ui-pickers/DatePicker';`.

Next error I get is:
```
C:/Development/cms/client/node_modules/material-ui-pickers/_shared/ModalDialog.d.ts
(3,23): Cannot find module '../../typings/utils'.
```

The referenced directory/file is not included in the npm package.
<img width="140" alt="material-ui-picker" src="https://user-images.githubusercontent.com/12628264/36934264-0c55f6b2-1ee7-11e8-953e-127eebb3f637.PNG">

## Possible solution

- Remove direct export of date utils from `index.js` and `index.d.ts`
- Include typings in npm package (e.g. move into src/)

## Live example

I'll be glad to provide a small github-repo if necessary.

**Comment by JohannesLamberts** [comment]
I created PR #238. 
The changes fixed the issues on my local machine.

**Comment by dmtrKovalenko** [comment]
Fixed in 1.0.0-rc.2
